seds/elasticsearch-sed/README.md

```diff
@@ -1,9 +1,9 @@
-This helm chart defines a Elasticsearch Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This helm chart defines a Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```

Comment:
```suggestion
This Helm chart defines an Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Suggestion: The H in Helm is always in capitals. Global comment. Please change in all such instances.